tests/ActiveFieldTest.php

```diff
@@ -55,7 +55,7 @@ public function testRadioList()
         $expectedHtml = <<<HTML
 <div class="form-group field-dynamicmodel-attributename">
 <label class="control-label">Attribute Name</label>
-<input type="hidden" name="DynamicModel[attributeName]" value=""><div id="dynamicmodel-attributename"><div class="radio"><label><input type="radio" name="DynamicModel[attributeName]" value="1"> name1</label></div>
+<input type="hidden" name="DynamicModel[attributeName]" value=""><div id="dynamicmodel-attributename" role="radiogroup"><div class="radio"><label><input type="radio" name="DynamicModel[attributeName]" value="1"> name1</label></div>
```

Comment:
[yiisoft/yii2#16713](https://github.com/yiisoft/yii2/pull/16713) changed the default output for radio lists